### PR TITLE
Feature/sassdocs

### PR DESCRIPTION
**Pull request by kirstyfisheye**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
     "marked-terminal": "~2.0.0",
     "postcss-reporter": "~3.0.0",
     "run-sequence": "~1.2.2",
-    "stylelint": "~7.7.1"
+    "sassdoc": "^2.1.20"
```
Comment (Igloczek) [inline package.json]: Missing comma. Also I'm using `~` for version targeting, not `^`.

File: task/sassdoc.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+	    theme.locale.forEach(locale => {
+	      
+	     	gulp.src(config.projectPath + theme.src + '/styles/**/*.scss')
+	      	.pipe(sassdoc(options));
+	    });
+
+	});
+
+};
```
Comment (Igloczek) [inline task/sassdoc.js]: Missing new line at end of file. Plase check CI tests output.

**Comment by Igloczek** [comment]
Some kind of description of this PR will be nice, I like to know what to merge.

Blank SASS theme doesn't support sassdocs out of the box, so it looks kinda pointless to add this task to Frontools, but I'm waiting for some convincing arguments (or maybe a way to create Frontools plugins, to easily extend Frontools, without modifying the core).

**Comment by kirstyfisheye** [comment]
Sorry, meant to add some notes straight away.

This is mostly for our internal use but I thought it might be of interest to others. We have a child theme that uses the Blank SASS theme as the parent. We need a quick, easy way to document what we add to our own theme because we have people editing our sites once they are live that may not have built them (i.e. support teams). 

If I've done it correctly, its been set up to only be created where you want it. For example our theme is called fisheye, so if we run `gulp sassdoc --theme fisheye` we only create docs for our child theme. If you just run `gulp sassdoc` it will create docs for every theme specified in themes.json

Please feel free to ignore if its of no use to you. We find the additional documentation really useful in helping maintain code quality across a team :)

**Comment by Igloczek** [comment]
Few things currently stop me to merge this PR:
- lack of sass-docs comments in "Blank" theme
- lack of support for inherited themes
- further maintenance

**Comment by Igloczek** [comment]
I'm going to close this, b/c IMO it didn't fit in the current shape of Frontools.

I started thinking about Frontools 2.0, which are probably going to be a Node.js package. It will let anybody to create a plugin extending Frontools core in very simple way, without changing anything in the core or asking for acceptance.
Stay tuned folks and develop own customizations in midtime! Fell free to share, even if I'll not be interested in merging it, it's a good way to understand what community need and make it possible in
 future releases 😉

**Pull request closed without merging**